main.go

```diff
@@ -17,13 +25,8 @@ import (
 	"github.com/nuts-foundation/nuts-admin/discovery"
 	"github.com/nuts-foundation/nuts-admin/identity"
 	"github.com/nuts-foundation/nuts-admin/issuer"
+	oidc2 "github.com/nuts-foundation/nuts-admin/oidc"
```

Comment:
bad package name alias (oidc2) I don't think the alias is even required, as there's no other import named "oidc".